stdlib/xml/dom/minidom.pyi

```diff
@@ -110,6 +116,7 @@ class Element(Node):
     schemaType: Any
     parentNode: Any
     tagName: str
+    nodeName: str
```

Comment:
Is this surely in the right class? I don't see anything that adds it to `Element`.

Reply:
Apparently I kept missing it because of this assignment: `self.tagName = self.nodeName = tagName`